Algebraic Topology/algebraic-topology.tex

```diff
@@ -80,7 +80,7 @@ \section{Introduction}
 (known as ``cycles'') and gap filling.
 
 \section{Reference}
-
+%
```

Comment:
I am not sure 1 notation deserves its own reference section. Maybe just state that in the definition?